<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainMenu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PackPresetsButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PACK PRESETS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PackTexturesButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PACK TEXTURES</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
